### PR TITLE
revise docs

### PR DESCRIPTION
**Pull request by xin-li-67**
## Motivation

Revise documents

## Modification

README.md
README_zh-CN.md

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,24 +36,25 @@ English | [简体中文](README_zh-CN.md)
...
+   - Supports training different tasks with a small amount of code, e.g., ImageNet can be trained with only 80 lines of code (400 lines of the original PyTorch example)
+   - Easily compatible with models from popular algorithm libraries such as TIMM, TorchVision, and Detectron2
 
-2. **Open architecture with unified interfaces**.
+2. **Open architecture with unified interfaces**:
 
-   - Handle different algorithm tasks with unified API, e.g., implement a method and apply it to all compatible models.
-   - Support different devices and hardwares with unified API, including CPU, GPU, IPU, Apple silicon, etc.
+   - Handles different algorithm tasks with unified APIs, e.g., implement a method and apply it to all compatible models.
+   - Provides a unified abstraction for the upper-level algorithm library, which supports various back-end devices such as Nvidia CUDA, Mac MPS, AMD, MLU, and other devices for model training.
```
Comment (C1rN09) [inline README.md:51]:
```suggestion
   - Provides a unified abstraction for upper-level algorithm libraries, which supports various back-end devices such as Nvidia CUDA, Mac MPS, AMD, MLU, and more for model training.
```

File: README_zh-CN.md [hunk 2]
```diff
@@ -38,19 +38,19 @@
 
 ## 简介
 
-MMEngine 是一个用于深度学习模型训练的基础库，基于 PyTorch，支持在 Linux、Windows、macOS 上运行。它具有如下三个亮点：
+MMEngine 是一个基于 PyTorch 用于深度学习模型训练的基础库，支持在 Linux、Windows、macOS 上运行。它具有如下三个亮点：
```
Comment (C1rN09) [inline README_zh-CN.md:41]: 我感觉“基于Pytorch”还是放在后面比较好？比较本质的内容放在前面

Comment (xin-li-67) [reply README_zh-CN.md:41]: 嗯嗯 理解了

**Review by C1rN09 — APPROVED** [review]
Good!

**Pull request merged**